### ISSUE TITLE
[WinUI 3 Gallery: Home]: Incorrect heading level is defined for 'Recently added samples' heading. 

### ISSUE THREAD

**Issue by sabareesanms** [microsoft/WinUI-Gallery#1955]
**Environment Details:**

**OS Version:** 24H2 (OS Build 26100.4315)

**App:** WinUI 3 Gallery

**App Version:** 2.6.0.0 (SDK 1.7)

**AT:** Screen reader

**Pre-Requisite:**

Turn on Narrator with shortcut key "win+ctrl+Enter" key.

**Repro Steps:**

1. Launch the WinUI 3 Gallery app.

2. Navigate to 'Home' list item present in left navigation pane and invoke it.

3. Now 'Home' page will be appeared

4. Navigate to 'Recently added samples' heading in headings mode.

5. Listen to the Screen reader announcement here.

**Actual Result:**

Incorrect heading level is defined for 'Recently added samples' heading text. On navigating to it in headings mode, Screen reader is announcing as "Recently added samples, heading level 1".

**Note:** Same issue is observed for 'Recently updatedsammples' heading

**Expected Result:**

Heading level 2 should be defined for 'Recently added samples' heading. On navigating to it Screen reader should announce as 'Recently added samples, heading level 2'.

**User Impact:**

Users with visual impairments who rely on screen reader will face difficulty if incorrect heading levels is defined for 'Recently added samples' heading.
